### DISCUSSION TITLE
Trouble binding to http port in docker container

### DISCUSSION THREAD

**Discussion by calikw** [jcorporation/myMPD#922] [Q&A] [answered]
Hi! Hoping for some help getting myMPD up and running in a Docker container on a Raspberry Pi 4.

I have the below environment variables set for my container:
```
TZ=America/Los_Angeles
UMASK_SET=022
MYMPD_SSL=false
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
MPD_HOST=192.168.132.2
MPD_PORT=6600
MYMPD_HTTP_PORT=8080
```

And yet, when I try to start the docker container, it appears to still be trying to bind to 0.0.0.0:80 for some reason:

```
NOTICE   mympd     Cache dir: "/var/cache/mympd"
NOTICE   mympd     Work dir: "/var/lib/mympd"
NOTICE   mympd     Config dir: "/var/lib/mympd/config"
NOTICE   mympd     Setting loglevel to NOTICE
NOTICE   mympd     Starting myMPD 10.1.4
b7099f5 1 mongoose.c:4201:mg_open_liste bind: 98
b7099f6 1 mongoose.c:3458:mg_listen     Failed: http://0.0.0.0:80, errno 98
EMERG    mympd     Can't bind to http://0.0.0.0:80
Exiting erroneous, thank you for using myMPD
```

Are there other config options I might need to set for my custom http port to be used?

Greatly appreciate any help anyone can provide!  Thank you!

**Comment by jcorporation** [comment, marked as answer]
Please read documentation:
https://jcorporation.github.io/myMPD/configuration/

>  After the first start all environment variables are ignored, except loglevel.
To change these settings afterwards, you must edit the files in the folder /var/lib/mympd/config/ and restart myMPD.

**Reply by calikw** [reply]
Ah, missed that part.  Thank you very much!
